app/controllers/accounts_controller.rb

```diff
@@ -68,20 +68,27 @@ def get_profile
     if @current_user
       user = User.find_by(id: params[:id].to_i)
 
-      if not user
+      unless user
         render(inertia: 'NotFound')
         return
       end
```

Comment:
usually these kind of checks can be done on a before_action method

Reply:
I started doing it ever since janeiro pointed the same thing out, just didn't go back and changed all of the others